src/source-af-packet.c

```diff
@@ -94,6 +94,8 @@
 #include <sys/mman.h>
 #endif
 
+#include <linux/net_tstamp.h>
```

Comment:
think it would be good to add a AC_CHECK_HEADERS test for this and include it conditionally